src/autocomplete/readMe.md

```diff
@@ -0,0 +1,38 @@
+## AutoComplete
+
+A basic autocomplete component.
+
+### Properties
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| `data` | `Array` | `Required` | An array of options for autocompletion |
```

Comment:
Default is required ? I feel default option should be N/A and required should be highlighted elsewhere.